index.html

```diff
@@ -1969,39 +2017,56 @@ <h2>The <dfn>NDEFRecordType</dfn> string</h2>
     </p>
   </section> <!-- visibility & focus: the suspended state -->
 
-  <section><h3>Aborting pending push operation</h3>
+  <section><h3>Clearing the pending push tuple</h3>
   <p>
-    To attempt to <dfn>abort a pending push operation</dfn> on an
+    To <dfn>clear the pending push tuple</dfn> on an
     <a>environment settings object</a>, perform the following steps:
     <ol class=algorithm>
       <li>
         If there is no <a>pending push tuple</a> |tuple|, abort these steps.
       </li>
       <li>
-        Let |writer:NFCWriter| be |tuple|'s writer.
+        Let |writer| be the <a>pending push tuple</a>'s writer.
+      </li>
+      <li>
+        Stop |writer|.[[\PushTimer]] if active.
+      </li>
+      <li>
+        Stop |writer|.[[\PushHandler]] if running.
```

Comment:
This is not possible... these steps are only executed when reading/writing and then we cannot abort it

Reply:
These are the clean-up steps, they are called to react to conditions after NFC operations, not during them.

Reply:
You can not stop a running 'push handler' it only runs when doing nfc operations. But as said in other comments, these are steps/algorithms and not something you can attach to an internal slot